.tekton/collector-component-pipeline.yaml

```diff
@@ -175,62 +160,62 @@ spec:
       operator: in
       values: [ "true" ]
     workspaces:
-    - name: output
-      workspace: workspace
     - name: basic-auth
       workspace: git-auth
 
   - name: determine-image-tag
     params:
-    - name: tag-suffix
+    - name: TAG_SUFFIX
       value: $(params.output-tag-suffix)
+    - name: SOURCE_ARTIFACT
+      value: $(tasks.clone-repository.results.SOURCE_ARTIFACT)
     runAfter:
-    # This task must run on a freshly cloned repository to prevent seeing any changes from other tasks.
-    - clone-repository
+      - clone-repository
```

Comment:
We have decided to remove all `runAfter` settings and let Tekton figure out the build graph based on the parameter dependencies (like `SOURCE_ARTIFACT` from `tasks.clone-repository.results.SOURCE_ARTIFACT` -> Tekton knows that this task need to run after `clone-repository`).

Please remove all runAfters from the pipeline file.